server/constants/preferences.go

```diff
@@ -0,0 +1,7 @@
+package constants
```

Comment:
0/5 since I haven't looked around this code enough to know where we'll need these, but I don't think we want a separate package just for constants. We can always move it when we find a home for it though

Reply:
I agree we can move it to the right place when we find it. For now I prefer to have it here, just as a way to keep things organized. Moving this around won't be a breaking change :stuck_out_tongue: